app/helpers/likes_helper.rb

```diff
@@ -0,0 +1,13 @@
+module LikesHelper
+  MAX_NAMES = 4.freeze
+
+  def tooltip(run)
+    count = run.likes.count
+
+    if count <= MAX_NAMES
+      return "Liked by #{run.likes.limit(MAX_NAMES).joins(:user).pluck(:name).to_sentence}"
+    end
+
+    return "Liked by #{run.likes.limit(MAX_NAMES - 1).joins(:user).pluck(:name).to_sentence} and #{count - (MAX_NAMES - 1)} others"
```

Comment:
Since it's only 4 names, why not take the `run.likes.limit(MAX_NAMES).joins(:user).pluck(:name)` and pull it out to a variable at the top, then use slicing of the array to only get the 3 names?  This would decrease line size for rubocop and let you reuse it for both branches.